packages/cyberstorm/src/components/Switch/Switch.tsx

```diff
@@ -8,16 +8,18 @@ export interface SwitchProps {
   state: boolean;
   onChange?: Dispatch<SetStateAction<boolean>>;
   disabled?: boolean;
+  labelId?: string;
```

Comment:
This should be named just `id`. It has other uses besides just coupling a label element. Also it's what developers would expect to use when wanting to add in id to an element.

Reply:
I was pondering if here should be specified what id it refers to. Thanks for clarifying it 👌